src/bidiMapper/domains/script/Realm.ts

```diff
@@ -171,7 +154,10 @@ export class Realm {
 
     if (deepSerializedValue.type === 'node') {
       if (Object.hasOwn(bidiValue, 'backendNodeId')) {
-        let navigableId = this.navigableId;
+        let navigableId = 'UNKNOWN';
+        if (this instanceof WindowRealm) {
```

Comment:
This line does not look right. The base class behavior depends on if the current instance of some specific derived  type.

Reply:
You can get back method `get navigableId()` returning `UNKNOWN` and override it in the `WindowRealm` instead. BTW, having the `UNKNOWN` navigable id  worth logging as warn.

Reply:
I'd rather not have unnecessary abstract methods. `instanceof` is meant for this purpose.